ui/js/directives/perf/compare.js

```diff
@@ -24,3 +24,17 @@ treeherder.directive(
             }
         };
     });
+
+treeherder.directive(
+    'revisionDescribe', function() {
```

Comment:
Let's call this directive revisionInformation and let's make it cover the whole revision header (see below).